locales/de.yml

```diff
@@ -1,76 +1,76 @@
-lang: German
-hello: Hello NoFWL
+lang: Deutsch
+hello: Hallo NoFWL
 window:
   message:
-    close: Do you want to exit the application when you click the [x] button?
+    close: Willst du die Anwendung schließen, wenn du [x] drückst?
 menu:
   app:
-    about: About NoFWL
-    check_update: Check for Updates
+    about: Über NoFWL
+    check_update: Auf Updates überprüfen
```

Comment:
`Auf Updates prüfen`?

Reply:
I thinks it's personal preference
`Nach Updates suchen`would also be correct